test/cli/fixtures/undefined-hooks/index.js

```diff
@@ -0,0 +1,6 @@
+QUnit.test( "undefined hooks", function( assert ) {
```

Comment:
As noted in the related issue (https://github.com/qunitjs/qunit/issues/1377#issuecomment-466259122), this reproduction can be smaller.

Additionally, this test description is pretty vague. I would prefer to see something to the effect of "assertion after errored/rejected tests does not cause further errors".

Reply:
This seems much better. Will update.